packages/p2p/src/components/dp2p-blocked/dp2p-blocked.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from 'react';
 import { Icon, Text } from '@deriv/components';
-import Dp2pBlockedChecklist from './dp2p-blocked-checklist.jsx';
-import Dp2pBlockedDescription from './dp2p-blocked-description.jsx';
-import { Localize } from '../i18next';
+import Dp2pBlockedChecklist from './dp2p-blocked-checklist';
+import Dp2pBlockedDescription from './dp2p-blocked-description';
+import { Localize } from 'Components/i18next';
 
 const Dp2pBlocked = () => {
     return (
-        <div className='dp2p-blocked' data-testid='dp2p-Dp2pBlocked_container'>
+        <div className='dp2p-blocked' data-testid='dt_dp2p-blocked-container'>
```

Comment:
```suggestion
        <div className='dp2p-blocked'>
```

I guess you can remove this since we don't use it in the test file :thinking:

Reply:
we still use it in verification component